rip/schema/list_field.py

```diff
@@ -30,6 +30,10 @@ def validate(self, request, value):
         if self.nullable and value is None:
             return ValidationResult(is_success=True)
 
+        if type(value) != list:
+            return ValidationResult(is_success=False,
+                                    reason="This field should be an array.")
```

Comment:
Can we add the name of the field in the error message, it is very much helpful while debugging/developing and also please change "array" to "list" it is more pythonic.